common/src/main/java/com/gentics/mesh/core/image/spi/AbstractImageManipulator.java

```diff
@@ -119,7 +118,6 @@ public Single<ImageInfo> readImageInfo(String path) {
 					bh.complete(toImageInfo(image));
 				}
 			} catch (Exception e) {
-				log.error("Reading image information failed", e);
```

Comment:
Remove duplications of error logging - it will be processed anyway in an async handler.